services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/AccessRequirementManagerImpl.java

```diff
@@ -232,9 +226,45 @@ public void deleteAccessRequirement(UserInfo userInfo,
 		verifyCanAccess(userInfo, accessRequirementId, ACCESS_TYPE.DELETE);
 		accessRequirementDAO.delete(accessRequirementId);
 	}
-	
+
 	private void verifyCanAccess(UserInfo userInfo, String accessRequirementId, ACCESS_TYPE accessType) throws UnauthorizedException, NotFoundException {
 		AuthorizationManagerUtil.checkAuthorizationAndThrowException(
 				authorizationManager.canAccess(userInfo, accessRequirementId, ObjectType.ACCESS_REQUIREMENT, accessType));
 	}
+
+	static AccessRequirement setDefaultValues(AccessRequirement ar) {
```

Comment:
My suggestion is to put this logic in the migration translator for the AR DBO. Also this manager can ensure that these fields are set upon creation or update.